android/app/src/main/java/org/fossasia/openevent/core/track/session/SessionsListAdapter.java

```diff
@@ -1,61 +1,43 @@
 package org.fossasia.openevent.core.track.session;
 
 import android.content.Context;
-import android.content.Intent;
-import android.graphics.Color;
-import android.graphics.PorterDuff;
 import android.support.v4.content.ContextCompat;
 import android.support.v7.widget.RecyclerView;
-import android.text.TextUtils;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
-import android.widget.ImageView;
-
-import com.amulyakhare.textdrawable.TextDrawable;
 
 import org.fossasia.openevent.R;
-import org.fossasia.openevent.common.ConstantStrings;
-import org.fossasia.openevent.common.date.DateConverter;
-import org.fossasia.openevent.common.date.DateService;
-import org.fossasia.openevent.common.notification.NotificationUtil;
-import org.fossasia.openevent.common.ui.Views;
-import org.fossasia.openevent.common.ui.WidgetUpdater;
 import org.fossasia.openevent.common.ui.base.BaseRVAdapter;
 import org.fossasia.openevent.common.utils.Utils;
-import org.fossasia.openevent.core.bookmark.BookmarkStatus;
 import org.fossasia.openevent.core.bookmark.OnBookmarkSelectedListener;
 import org.fossasia.openevent.data.Session;
-import org.fossasia.openevent.data.Speaker;
-import org.fossasia.openevent.data.Track;
 import org.fossasia.openevent.data.repository.RealmDataRepository;
-import org.threeten.bp.ZonedDateTime;
 
-import java.util.ArrayList;
 import java.util.List;
 
-import io.reactivex.Observable;
 import io.reactivex.disposables.CompositeDisposable;
-import timber.log.Timber;
-
-import static org.fossasia.openevent.core.bookmark.BookmarkStatus.Status.CODE_ERROR;
-import static org.fossasia.openevent.core.bookmark.BookmarkStatus.Status.CODE_UNDO_ADDED;
 
+/**
+ * User: MananWason
+ * Date: 26-06-2015
+ */
 public class SessionsListAdapter extends BaseRVAdapter<Session, SessionViewHolder> {
 
-    private final Context context;
-    public static int listPosition;
+    private Context context;
     private int type;
-    private static final int locationWiseSessionList = 1;
-    private static final int trackWiseSessionList = 4;
-    private static final int speakerWiseSessionList = 2;
     private OnBookmarkSelectedListener onBookmarkSelectedListener;
+    private HandleItemClicks handleItemClicks;
 
     private RealmDataRepository realmRepo = RealmDataRepository.getDefaultInstance();
     private final CompositeDisposable compositeDisposable = new CompositeDisposable();
 
     private int color;
 
+    public interface HandleItemClicks {
+        void itemOnClick(int adapterPosition, int layoutPosition);
+    }
```

Comment:
Actually, this is not helpful. Interface names should be adjectives, adverbs or nouns describing what implementer will be doing. 

Like BlahBlahBlahListener.

It can simply be `OnItemClickListener`